extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,61 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** Inline attachment specifications */
+.i-amphtml-story-inline-page-attachment-chip {
+  display: flex !important;
```

Comment:
Might not need this since we're using margin to center `i-amphtml-story-inline-page-attachment-arrow`.

Reply:
they become a single column instead of a row without it